tests/factories/transcript.py

```diff
@@ -10,20 +10,17 @@ class Meta:
 
     video_id = Sequence(lambda n: f"video_id_{n}")
     transcript_id = Sequence(lambda n: f"transcript_id_{n}")
-    transcript = [
-        {
-            "text": "[Music]",
-            "start": 0.0,
-            "duration": 7.52,
-        },
-        {
-            "text": "how many of you remember the first time",
-            "start": 5.6,
-            "duration": 4.72,
-        },
-        {
-            "text": "you saw a playstation 1 game if you were",
-            "start": 7.52,
-            "duration": 4.72,
-        },
-    ]
+    transcript = Sequence(
+        lambda n: [
+            {
+                "text": f"This is the first line of transcript {n}",
+                "start": 0.0,
+                "duration": 7.52,
+            },
+            {
+                "text": f"This is the second line of transcript {n}",
+                "start": 5.6,
+                "duration": 4.72,
+            },
+        ]
+    )
```

Comment:
Fixed `TranscriptFactory` so that each `Transcript` object has a different value for `Transcript.transcript`. This is needed by the tests below, and is probably a good idea in general: `TranscriptFactory` always returning `Transcript`'s with the same `Transcript.transcript` is a bit of a footgun (tests that compare the values of two `Transcript.transcript`'s could have false positives thinking they came from the same `Transcript` when in fact they belonged to different ones).